inv/menu.go

```diff
@@ -60,13 +69,16 @@ func UpdateMenu(p *player.Player, m Menu) {
 	sendMenu(p, m, true)
 }
 
+// sendMenu sends the menu to a player.
 func sendMenu(p *player.Player, m Menu, update bool) {
 	s := player_session(p)
 
-	inv := inventory.New(m.container.Size(), func(slot int, before, after item.Stack) {})
-	inv.Handle(handler{p: p, menu: m})
-	for i, it := range m.items {
-		_ = inv.SetItem(i, it)
+	var inv *inventory.Inventory
```

Comment:
this doesn't look right, shouldn't it be 

```go
inv := m.inventory

if m.custom {
// handle
}

Reply:
no if its custom theres no preset handler

Reply:
if its not custom then the handler is set